Packs/Edgescan/Integrations/Edgescan/README.md

```diff
@@ -0,0 +1,5110 @@
+Cloud-based continuous vulnerability management and penetration testing solution.
+This integration was integrated and tested with version xx of Edgescan
+## Configure Edgescan on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Edgescan.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Server URL |  | True |
+    | Use system proxy settings |  | False |
+    | Trust any certificate (not secure) |  | False |
+    | API Key |  | True |
+    | Max number of incidents to fetch at once |  | False |
+    | Fetch vulnerabilities with CVSS | This has to be an exact value. No filter operator available. | False |
+    | Fetch vulnerabilities with risk more than |  | False |
+    | First fetch time | How many days to fetch back on first run. | False |
+    | Fetch incidents |  | False |
+    | Incident type |  | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### edgescan-host-get-hosts
+***
+Get a list of all hosts
+
+
+#### Base Command
+
+`edgescan-host-get-hosts`
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Edgescan.HostGetHosts | Unknown | List of all hosts | 
+
+
+#### Command Example
+```!edgescan-host-get-hosts```
+
+#### Human Readable Output
+
+>### Hosts
+>**No entries.**
+
+
+### edgescan-host-get
+***
+Get detailed information about a host.
+
+
+#### Base Command
+
+`edgescan-host-get`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | The host id. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Edgescan.HostGet | Unknown | Detailed host information | 
+
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
```

Comment:
Please remove all the empty sections in the file

Reply:
done